README.md

```diff
@@ -0,0 +1,37 @@
+# keyboard-only-outlines
+
+Disable outlines caused by navigation methods other than tab-navigation.
+
+## Installation
+
+```
+$ npm install keyboard-only-outlines
+```
+
+## Usage
+
+```
+import keyboardOnlyOutlines from 'keyboard-only-outlines'
+
+const dispose = keyboardOnlyOutlines()
+/*
+* dispose will be a function which deactivates this behaviour and can be called at any time
+*/
+```
+
+The function may also be invoked with the following options: 
+
+```
+keyboardOnlyOutlines({styles: ..., styleSheetParent: ...})
```

Comment:
Missing `js` as well. Also add spaces between the brackets and the keys of the object.

Reply:
Missing js in the function name? Did you want to call the function `jsKeyboardOnlyOutlines`?